plugins/parsers/json/parser.go

```diff
@@ -219,6 +222,8 @@ func (p *Parser) Parse(buf []byte) ([]telegraf.Metric, error) {
 		return p.parseObject(v, timestamp)
 	case []interface{}:
 		return p.parseArray(v, timestamp)
+	case nil:
+		return make([]telegraf.Metric, 0), nil
```

Comment:
I think we should be able to return nil here
```suggestion
		return nil, nil
```

Reply:
Updated, thanks!